Makefile

```diff
@@ -1,17 +1,45 @@
 TARGET_DIR?=$(shell pwd)
+PROTO_DIR=proto
+SRC_MAIN_PROTO_DIR=src/main/proto
 GITHUB_GROUP=github.com/th2-net
 
+TH2_GRPC_COMMON=th2-grpc-common
+TH2_GRPC_COMMON_URL=$(GITHUB_GROUP)/$(TH2_GRPC_COMMON)@makefile
+
 TH2_COMMON_GO=th2-common-go
 TH2_COMMON_GO_URL=$(GITHUB_GROUP)/$(TH2_COMMON_GO)@rabbitMQ_dev
 
-clean-deps:
-	-rm go.work
+MODULE_NAME=th2-grpc
+MODULE_DIR=$(TARGET_DIR)/$(MODULE_NAME)
+
+configure-go:
+	go install google.golang.org/protobuf/cmd/protoc-gen-go@latest
+	go install google.golang.org/grpc/cmd/protoc-gen-go-grpc@latest
+
+clean-dependencies:
+	-rm -rf $(PROTO_DIR)
 
-deps: clean-deps 
-	go work init .
+prepare-dependencies: clean-dependencies
+	mkdir $(PROTO_DIR)
+	mkdir $(TH2_COMMON_GO)
 
+	go get -u -t $(TH2_GRPC_COMMON_URL)
 	go get -u -t $(TH2_COMMON_GO_URL)
-	sleep 4
-	go list -m -f '{{.Dir}}' $(TH2_COMMON_GO_URL)
-	ls $(shell go list -m -f '{{.Dir}}' $(TH2_COMMON_GO_URL))
-	@cd $(subst \,/, $(shell go list -m -f '{{.Dir}}' $(TH2_COMMON_GO_URL))) && make deps TARGET_DIR=$(TARGET_DIR)
\ No newline at end of file
+
+	sleep 1
+	cp -r --no-preserve=mode,ownership $(subst \,/, $(shell go list -m -f '{{.Dir}}' $(TH2_GRPC_COMMON_URL))/$(SRC_MAIN_PROTO_DIR)/*) $(PROTO_DIR)
+	cp -r --no-preserve=mode,ownership $(subst \,/, $(shell go list -m -f '{{.Dir}}' $(TH2_COMMON_GO_URL))/*) $(TH2_COMMON_GO)
```

Comment:
This is redundant because common-go has been already included via `go get -u -t $(TH2_COMMON_GO_URL)`